packages/core/src/tools/display.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Keep references on console methods to avoid triggering patched behaviors
+ */
+export const display: Pick<typeof console, 'log' | 'warn' | 'error'> = {
+  log: console.log,
+  warn: console.warn,
+  error: console.error,
```

Comment:
I don't think this is enough to ensure that we are using unpatched functions in this kind of setup:

```
    <script src="http://localhost:8080/datadog-rum-recorder.js"></script>
    <script>
      DD_RUM.init({
        clientToken: 'xxx',
        applicationId: 'xxx',
      })
    </script>
    <script src="http://localhost:8080/datadog-logs.js"></script>
    <script>
      DD_LOGS.init({
        clientToken: 'xxx',
      })
    </script>
```

Reply:
Agree, I am not sure that this is an issue though.
Wort case scenario, a rum displayed error generate a logs error or vice versa, so we should not end up in infinite loop.
wdyt?

Reply:
Ok, could you adjust the comment to make it clear that it can happen then?